views/pages/bucketlist/bucketlistOverzicht.ejs

```diff
@@ -6,7 +6,24 @@
     <p>Choose your bucketlist locations</p>
 
     <!-- Form with post method including 10 locations -->
-    <form id="location-form" method="POST" action="/bucketlistResultaat">
+    <form id="location-form" method="POST" action="/bucketlist">
+        <label>
+            Name
+            <input type="text" name="name">
+        </label> <br>
```

Comment:
Please refrain from using using `<br>` (a break rule) element to get a specific look we want. We can do a lot of magic with SCSS/CSS ✨ 🎉  
If the form looks a bit wonky for now, that's okay. We're going for functionality after all.